next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx

```diff
@@ -0,0 +1,262 @@
+
+The [IOTA SDK](https://github.com/iotaledger/iota-sdk) is a brand new project which is a combination and improvement of both [iota.rs](https://github.com/iotaledger/iota.rs) and [wallet.rs](https://github.com/iotaledger/wallet.rs).
+If you were already supporting Shimmer with wallet.rs, we encourage you to switch to the SDK as wallet.rs will be deprecated and not receive any updates anymore.
+
+Most of the APIs are the same and the changes are fairly minimal, here are some of the major changes:
+
+- The npm package changes from `@iota/wallet` to `@iota/sdk`
+- `AccountManager` becomes `Wallet`
+- `manager.generateMnemonic()` becomes `Utils.generateMnemonic()`
+- `generateAddress` becomes `generateEd25519Addresses`
+- `sendAmount` becomes `send`
+- The callbacks to `listen` now provide a properly typed `Event`
+
+</details>
+
+### 1. Set Up the IOTA SDK
+
+First, you should install the components needed to use the IOTA SDK and the binding of your choice; it may vary a bit from language to language. In the case of the [NodeJs binding](../getting-started/nodejs.mdx), it is straightforward since it is distributed via the `npm` package manager.
+
+You can read more about [backup and security in this guide](https://wiki.iota.org/shimmer/introduction/how_tos/backup_security).
+
+```bash
+npm install @iota/sdk dotenv
+```
+
+:::warning
+
+We use `dotenv` to share user-defined constants across many examples. This is for convenience only and shouldn't be done in production.
+
+:::
+
+You can create a `.env` by running the following command:
+
+```bash
+touch .env
+```
+
+The following examples will need these variables to exist, here are some default values you can use but also change:
+
+```bash
+WALLET_DB_PATH="./wallet-database"
+NODE_URL="https://api.testnet.shimmer.network"
+STRONGHOLD_SNAPSHOT_PATH="./wallet.stronghold"
+STRONGHOLD_PASSWORD="*K6%79#yeFn7AMQhMQ"
+EXPLORER_URL="https://explorer.shimmer.network/testnet"
+```
+
+#### 1 Generate a mnemonic
+
+```js reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/exchange/0-generate-mnemonic.ts
+```
+
+After you have updated the `.env` file, you can initialize the `Wallet` instance with a secret manager([`Stronghold`](https://wiki.iota.org/stronghold.rs/welcome) by default) and client options.
+
+:::note
+
+Manage your password with the utmost care.
+
+:::
+
+By default, the Stronghold file will be called `wallet.stronghold`. It will store the seed (derived from the mnemonic) that serves as a cryptographic key from which all accounts and related addresses are generated.
+
+One of the key principles behind the `stronghold` is that no one can get a seed out of it, so you should also back up your 24-word mnemonic in a secure place because there is no way to recover it from the `.stronghold` file. You deal with accounts using the `Wallet` instance exclusively, where all complexities are hidden under the hood and are dealt with securely.
```

Comment:
```suggestion
One of the key principles behind `Stronghold` is that is impossible to extract a seed from it ever again, that is, recover it from the `.stronghold` file. That's why you should always back up your 24-word mnemonic and have it stored in safe place. You deal with accounts using the `Wallet` instance exclusively, which abstracts away all internal complexities and makes transacting in a network very easy and convenient.
```

Reply:
This sentence is not factually correct. It is possible to get the seed out of stronghold.

Reply:
Should probably remove that whole part

Reply:
Replace seed with mnemonic and it's correct :trollface:

Reply:
I was thinking the exact same thing ahah